src/gui_common/tooltip/ToolTipManager.cs

```diff
@@ -516,4 +519,54 @@ private void HideAllToolTips()
             }
         }
     }
+
+    private void CreateOrganelleTooltips()
+    {
+        var packedTooltip = GD.Load<PackedScene>("res://src/microbe_stage/editor/tooltips/SelectionMenuToolTip.tscn");
+
+        foreach (var organelle in SimulationParameters.Instance.GetAllOrganelles())
+        {
+            var tooltip = packedTooltip.Instantiate<SelectionMenuToolTip>();
+
+            Instance.AddToolTip(tooltip, "organelleSelection");
+
+            tooltip.Description = organelle.Description;
+            tooltip.MutationPointCost = organelle.MPCost;
+            tooltip.Name = organelle.InternalName;
+            tooltip.DisplayName = organelle.Name;
+            tooltip.RequiresNucleus = organelle.RequiresNucleus;
+            tooltip.ThriveopediaPageName = organelle.InternalName;
+            tooltip.ProcessesDescription = organelle.ProcessesDescription == null ?
+                string.Empty :
+                organelle.ProcessesDescription;
+
+            if (organelle.Components.Storage != null)
+            {
+                tooltip.AddModifierInfo(new LocalizedString("STORAGE").ToString(),
```

Comment:
I'll note that thanks to this, this other issue will be solved by this PR: https://github.com/Revolutionary-Games/Thrive/issues/2306